### DISCUSSION TITLE
composer bump --dev

### DISCUSSION THREAD

**Discussion by staabm** [composer/composer#12249] [General]
I wonder what you think about adding a `--dev` switch to `composer bump` so one can easily bump only require-dev stuff.

IMO require-dev dependencies are a different beast as regular require dependencies, because they usually can be bumped more often as they should not affect the consumers of a package

**Comment by stof** [comment]
This flag already exists, except that it is named `--dev-only`

**Reply by staabm** [reply]
ohh I see. maybe we just need a better error message then :-)

![grafik](https://github.com/user-attachments/assets/add1ce40-8223-41c6-9719-aeb539f9a6f3)

**Reply by staabm** [reply]
I guess we are talking about symfony console default handling.. but looking at the screenshot I just sent makes me realize that it actually includes this information.

sorry for the noise and thanks for the help - I guess its just me beeing not yet used to this command ;)
